src/components/Table/TableHead/_table-head.scss

```diff
@@ -2,10 +2,14 @@
 @import '~carbon-components/scss/globals/scss/layout';
 
 .#{$prefix}--data-table {
```

Comment:
Does this need to be scoped to only be applied when lightweight  class is applied?